### DISCUSSION TITLE
"Your game Volume is too low" despite not being too low

### DISCUSSION THREAD

**Discussion by nyannoying1337** [ppy/osu#18058] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/17437302/166290632-6f8d2406-539e-4219-8929-35e29120bf24.png)

While this does might seem rather low, its not. Is there an option to hide the this notification?

**Comment by bdach** [comment]
You are right at the threshold here:

https://github.com/ppy/osu/blob/4442ed89b59d57bba6b0e588ef270cda036cb116/osu.Game/Screens/Play/PlayerLoader.cs#L505

It still seems a bit weird to me that 5% master volume is apparently playable for you. What's the audio situation there?

**Reply by nyannoying1337** [reply]
im using hyperx cloud 2 ( headphones ). 

if thats right at the threshold tho i can just put it at 6 master volume. i still find it rather odd to get told "your volume is too low to hear anything" despite playing like this for years

**Reply by frenzibyte** [reply]
Would really suggest increasing your master volume a touch and avoid this (10% maybe). 5% is quite unplayable for a rhythm game, and no one has reported such issue since the time it was changed (which was somewhere around months ago).

If you find it playable, I would question whether your local audio setup is properly balanced, actually.

**Reply by RevertedClock** [reply]
if i may add my opinion here. as someone who is using a hearing aid thats pretty much what i use all be it with Music and effect at like 70%. while i could take them out and set the master volume higher its a pain to take them out for a small osu! session. i think overall youre right that it should be higher but i also think you should give us the option to turn that "reminder" off because overall its us playing and what we think is better for us and all

**Reply by nyannoying1337** [reply]
> Would really suggest increasing your master volume a touch and avoid this (10% maybe). 5% is quite unplayable for a rhythm game, and no one has reported such issue since the time it was changed (which was somewhere around months ago).
> 
> If you find it playable, I would question whether your local audio setup is properly balanced, actually.

well i put it to 6% and that gets rid of it as well so its solved id say. its just something ive been doing for years and i have been fine 


What i would REALLY suggest however is not restoring the volume to "MAX" whenever you click on that notifcation because thats just crazy @frenzibyte

**Reply by RevertedClock** [reply]
> What i would REALLY suggest however is not restoring the volume to "MAX" whenever you click on that notifcation because thats just crazy @frenzibyte

That i 100% agree with. the amount of times i earraped me with that is just stupid

**Reply by frenzibyte** [reply]
Well, you have shifted the scope of this thread completely to a separate issue, but fine. This is tracked in https://github.com/ppy/osu/discussions/16190.

**Reply by peppy** [reply]
You should be adjusting your SYSTEM (windows) volume so that you don't need to set each application to negligible volume levels.

Please try doing this at a windows level:

**Reply by KaffeinatedKat** [reply]
This notification should be removed, and replaced with a "Your game is muted" if the volume is at 0%. I have my system volume set pretty high and I set all my applications volume to my liking because its easier for me, and this notification is just stupid imho

**Reply by peppy** [reply]
Do you set each application on your system to 1% volume manually? What happens when you start a new application? Does it not deafen you?

**Reply by plaguemask** [reply]
Adding my own... *particular*... experience:
Most of the time I use bluetooth with my headset, but when playing osu! I plug it in. Because of the convoluted ways in which my headset's volume setting does (and doesn't) interact with Windows's volume setting, the effective volume becomes much louder after plugging it in. Rather than adjusting the headset's volume before and after playing the game every time, I'd rather just set osu!'s volume rather low to compensate, often under 5%.

**Reply by AuracleTech** [reply]
Im currently on a conference call with my boss, we're in pause so I want to play osu at 1-5% volume but still listen in case the pause is finished.

I don't want to change the setting from withing windows as it's easier to change it inside osu.
Muting the notification could be nice...

**Reply by peppy** [reply]
You know the notification only shows once and can be easily dismissed, correct?

**Reply by AuracleTech** [reply]
Yeah but it's annoying considering I get it multiple times a day and every day

**Reply by peppy** [reply]
What's the annoying part? That it stays on the screen too long? Would it help if it only stayed for 2.5 seconds instead of 12?

**Reply by AuracleTech** [reply]
It would be better yes, but why not just add an option for it?

**Reply by peppy** [reply]
Because more options are not the answer to everything.

**Comment by peppy** [comment, marked as answer]
This should be improved by https://github.com/ppy/osu/pull/20379.
